index.bs

```diff
@@ -900,10 +917,10 @@ partial interface HTMLIFrameElement {
 
     <div class="algorithm"
     data-algorithm="define-inherited-policy-in-container">
-    Given a feature (|feature|), null or a <a>navigable container</a>
-    (|container|), and an <a>origin</a> for a document in that container
-    (|origin|), this algorithm returns the <a>inherited policy</a> for that
-    feature.
+    Given a [=feature=] (|feature|), null or a <a>navigable container</a>
+    (|container|), and an <a for="Document">origin</a> for a {{Document}} in
+    that container (|origin|), this algorithm returns the <a for=/>inherited
```

Comment:
Yeah I think this goes to the larger question of whether or not we need a `for=/` dfn for "inherited policy" in the first place anyways, now that this PR gives the `for="permissions policy"` definition of "inherited policy".

But yeah for this specific comment, it's probably not right either way. I think this should use "inherited policy for a feature" since that's really what we're referencing. I'll change this.